pennylane/qaoa/cycle.py

```diff
@@ -433,13 +432,12 @@ def _square_hamiltonian_terms(
     for (coeff1, op1), (coeff2, op2) in products:
         squared_coeffs.append(coeff1 * coeff2)
 
-        if isinstance(op1, qml.Identity):
+        if isinstance(op1, Identity):
```

Comment:
:thinking: what?

Reply:
Oh :hankey: here the nasty codefactor came again

Reply:
Pylint takes priority over codefactor. If it's passing, I think we're fine.